### PR TITLE
Add tests for Event.srcElement and Event.returnValue

### PR DESCRIPTION
**Pull request by miketaylr**
(these tests were already r+'d in https://github.com/w3c/web-platform-tests/pull/4790, but splitting them out from the `window.event` stuff so they can land independent of that).

r? @annevk

### CODE REVIEW

**Comment by w3c-bots** [comment]
# Build PENDING

Started: None
Finished: None





View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/26698)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/360437814)

File: dom/interfaces.html [hunk 1]
```diff
@@ -8,6 +8,7 @@
 
 <h1>DOM IDL tests</h1>
 <div id=log></div>
+
```
Comment (annevk) [inline dom/interfaces.html]: Maybe not touch this?

**Review by annevk — APPROVED** [review]
Seems there is some duplication, but generally seems fine.

**Comment by ahmadmiz** [comment]
Dh

**Comment by annevk** [comment]
Addressed my own nit.

**Comment by annevk** [comment]
Hmm, these tests cause harness failures due to not wrapping the listeners appropriately.

**Pull request merged**